target/linux/ipq40xx/files-4.14/arch/arm/boot/dts/qcom-ipq4019-linksys_ea8300.dts

```diff
@@ -0,0 +1,394 @@
+					};
+
+					partition@6280000 {
+						label = "alt_rootfs";
+						reg = <0x6280000 0x5500000>;
+					};
+
+					partition@b780000 {
+						label = "sysdiag";
+						reg = <0xb780000 0x100000>;
+						read-only;
+					};
+
+					partition@b880000 {
+						label = "syscfg";
+						reg = <0xb880000 0x4680000>;
+						read-only;
+					};
+				};
+			};
+		};
+
+
+		mdio@90000 {			// Enables phys 0-4
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+
+		wifi@a000000 {			// wifi0
+			status = "okay";
+			qcom,ath10k-calibration-variant = "linksys-ea8300-fcc";
+		};
+
+		wifi@a800000 {			// wifi1
+			status = "okay";
+			qcom,ath10k-calibration-variant = "linksys-ea8300-fcc";
+		};
+
+		pci@40000000 {			// pcie0
+			status = "okay";
+
+			perst-gpio = <&tlmm 38 GPIO_ACTIVE_LOW>;
+			wake-gpio = <&tlmm 50 GPIO_ACTIVE_LOW>;
+
+			bridge@0,0 {
+				reg = <0x00000000 0 0 0 0>;
+				#address-cells = <3>;
+				#size-cells = <2>;
+				ranges;
+
+				wifi2: wifi@1,0 {
+					compatible = "qcom,ath10k";
+					status = "okay";
+					reg = <0x00010000 0 0 0 0>;
+					qcom,ath10k-calibration-variant = "linksys-ea8300-fcc";
+				};
+			};
+		};
+
+
+		ssphy@9a000 {			// usb3_ss_phy
+			status = "okay";
+		};
+
+		hsphy@a6000 {			// usb3_hs_phy
+			status = "okay";
+		};
+
+		usb3@8af8800 {
+			status = "okay";
+		};
+
+		hsphy@a8000 {			// usb2_hs_phy
+			status = "okay";
+		};
+
+		usb2@60f8800 {
+			status = "okay";
+		};
+
+
+
+		tcsr@1949000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1949000 0x100>;
+			qcom,wifi_glb_cfg = <TCSR_WIFI_GLB_CFG>;
+		};
+
+		tcsr@194b000 {
+			compatible = "qcom,tcsr";
+			reg = <0x194b000 0x100>;
+			qcom,usb-hsphy-mode-select = <TCSR_USB_HSPHY_HOST_MODE>;
+		};
+
+		ess_tcsr@1953000 {  // May not be needed for ipqess / qca8k
+			compatible = "qcom,tcsr";
+			reg = <0x1953000 0x1000>;
+			qcom,ess-interface-select = <TCSR_ESS_PSGMII>;
+		};
+
+		tcsr@1957000 {
+			compatible = "qcom,tcsr";
+			reg = <0x1957000 0x100>;
+			qcom,wifi_noc_memtype_m0_m2 = <TCSR_WIFI_NOC_MEMTYPE_M0_M2>;
+		};
+
+
+		dma@7884000 {			// blsp_dma
+			status = "okay";
+		};
+
+		dma@7984000 {			// qpic_bam
+			status = "okay";
+		};
+
+		dma@8e04000 {			// cryptobam
+			status = "okay";
+		};
+
+		edma@c080000 {
+			status = "okay";
+		};
+
+
+		pinctrl@1000000 {		// tlmm
+
+			serial_0_pins: serial0-pinmux {
+				pins = "gpio16", "gpio17";
+				function = "blsp_uart0";
+				bias-disable;
+			};
+
+			i2c_0_pins: i2c-0-pinmux {
+				pins = "gpio20", "gpio21";
+				function = "blsp_i2c0";
+				bias-disable;
+			};
+
+			nand_pins: nand-pins {
+				pins = "gpio53", "gpio55", "gpio56",
+				       "gpio57", "gpio58", "gpio59",
+				       "gpio60", "gpio62", "gpio63",
+				       "gpio64", "gpio65", "gpio66",
+				       "gpio67", "gpio68", "gpio69";
+				function = "qpic";
+                        };
+		};
+
+
+		i2c@78b7000 {			// blsp1_i2c3
+			status = "ok";
```

Comment:
Please use "okay". As "ok" is deprecated.

However, I have a question: Is the i2c connected to anything at all?
If not please leave it disabled (by removing this i2c node altogether)